cf-plugin-backup-master.yaml

```diff
@@ -54,6 +62,13 @@ resources:
     <<: *s3-configs
     regexp: ((s3-prefix))cf-plugin-backup-(.*windows-386.*)\.tgz
 
+- name: src-release
+  type: github-release
+  source:
+    owner: ((src-owner))
+    repository: ((src-repo))
+    access_token: ((github-access-token))
```

Comment:
Do we want to add `drafts: true` for now, and make the actual releasing manual? So we can add release notes (until we automate that too), etc.?

Reply:
Makes sense. The release notes are very much a sparse thing.